code/modules/research/designs/machine_designs.dm

```diff
@@ -189,7 +189,7 @@
 	name = "Machine Design (DNA Scanner)"
 	desc = "Allows for the construction of circuit boards used to build a Cloning Scanner."
 	id = "dnascanner"
-	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
```

Comment:
should medical flag even stay there

Reply:
If this is wanted, the gene sequencer would also have to be moved, but I see no issue leaving them in both med and sci. I have no suggestions on either, unless someone really wants them moved.